tests/app/notifications/rest/test_callbacks.py

```diff
@@ -69,21 +69,42 @@ def test_dvla_callback_autoconfirm_does_not_call_update_letter_notifications_tas
     assert not update_task.called
 
 
-def test_dvla_callback_calls_update_letter_notifications_task(client, mocker):
+def test_dvla_callback_calls_does_not_update_letter_notifications_task_with_invalid_file_type(client, mocker):
     update_task = \
         mocker.patch('app.notifications.notifications_letter_callback.update_letter_notifications_statuses.apply_async')
-    data = _sample_sns_s3_dvla_response_callback()
+
+    data = _sample_sns_s3_callback("bar.txt")
+    response = dvla_post(client, data)
+
+    assert response.status_code == 200
+    assert not update_task.called
+
+
+def test_dvla_rs_txt_file_callback_calls_update_letter_notifications_task(client, mocker):
+    update_task = \
+        mocker.patch('app.notifications.notifications_letter_callback.update_letter_notifications_statuses.apply_async')
+    data = _sample_sns_s3_callback('Notify-20170411153023-rs.txt')
+    response = dvla_post(client, data)
+
+    assert response.status_code == 200
+    assert update_task.called
+    update_task.assert_called_with(['Notify-20170411153023-rs.txt'], queue='notify-internal-tasks')
+
+
+def test_dvla_rsp_txt_file_callback_calls_update_letter_notifications_task(client, mocker):
+    update_task = \
+        mocker.patch('app.notifications.notifications_letter_callback.update_letter_notifications_statuses.apply_async')
+    data = _sample_sns_s3_callback('NOTIFY.20170823160812.RSP.TXT')
     response = dvla_post(client, data)
 
     assert response.status_code == 200
     assert update_task.called
-    update_task.assert_called_with(['bar.rs.txt'], queue='notify-internal-tasks')
 
```

Comment:
Are we missing an assert on the call args for the `RSP.TXT` test, probably got lost during the rebase?

Reply:
Now added back in